wordpress-mobile/gutenberg-mobile
# Fix merge of preformatted blocks

Fixes #18476 for mobile RN

GB PR: https://github.com/WordPress/gutenberg/pull/18809

To test:
 - Add two preformatted blocks
 - Write several lines on each
 - Merge them
 - See if whitespace is kept properly


Update release notes:
